app/src/lib/errors.ts

```diff
@@ -9,3 +9,10 @@ export class ApiError extends Error {
     this.status = status; // Set the status
   }
 }
+
+export class PacketNameError extends Error {
```

Comment:
can be removed

Reply:
What error would you pass to ErrorComponent instead?

Reply:
oh yip you right